docs/modules/databases/neo4j.md

```diff
@@ -126,6 +126,26 @@ public class ExampleTest {
 }
 ```
 
+### Add Neo4j Docker Labs plugins
+
+Add any Neo4j Labs plugin from the [Neo4j Docker Labs plugin list](https://neo4j.com/docs/operations-manual/4.4/docker/operations/#docker-neo4jlabs-plugins).
+
+!!! note
+    At the moment only the plugins available in Neo4j Docker 4.4 are supported by type.
+    If you want to register another support Neo4j Labs plugin, you have to add the environment manually,
+    e.g. `withEnv("NEO4JLABS_PLUGINS", "[\"anotherPlugin\"]")`.
+    Please refer to the list of [supported Docker image plugins](https://neo4j.com/docs/operations-manual/current/docker/operations/#docker-neo4jlabs-plugins).
+
+```java
+@Testcontainers
+public class ExampleTest {
+
+    @Container
+    Neo4jContainer neo4jContainer = new Neo4jContainer()
+        .withLabsPlugin(Neo4jLabsPlugin.APOC);
+}
+```
```

Comment:
Please add code in docs as code includes from the tests (see docs of other modules on how to do it).